pinot-controller/src/test/java/org/apache/pinot/controller/helix/ControllerTest.java

```diff
@@ -500,31 +515,32 @@ protected void stopFakeInstance(String instanceId) {
    */
   protected void addSchema(Schema schema)
       throws IOException {
-    String url = _controllerRequestURLBuilder.forSchemaCreate();
-    PostMethod postMethod = sendMultipartPostRequest(url, schema.toSingleLineJsonString());
-    assertEquals(postMethod.getStatusCode(), 200);
+    getControllerRequestClient().addSchema(schema);
   }
 
   protected Schema getSchema(String schemaName) {
-    Schema schema = _helixResourceManager.getSchema(schemaName);
-    assertNotNull(schema);
-    return schema;
+    try {
```

Comment:
Suggest not changing this one because all the getters are not using http request right now (e.g. `getOfflineTableConfig` and `getRealtimeTableConfig`). We may change them in a separate PR if necessary.